app/styles/_map.scss

```diff
@@ -1,9 +1,201 @@
 .map__container {
   width: 100%;
   height: 100%;
+  position: relative;
+}
+
+.menubar {
+  background-color: lighten($lightest-grey, 5%);
+  padding: .7em 0;
+  box-shadow: 0px 1px 6px rgba(0, 0, 0, .5);
+  li {
+    display: inline-block;
+    margin-right: 2.5em;
+  }
+  label {
+    color: $grey;
+    font-weight: $base-font-bold;
+    margin-right: .6em;
+  }
 }
 
 .disabled {
-  opacity: 0.4;
+  /*opacity: 0.4;*/
   pointer-events: none;
 }
+
+.button {
+  -webkit-border-radius: 4px; 
+  -moz-border-radius: 4px; 
+  border-radius: 4px;
+  padding: .3em 1.5em .15em;
+  letter-spacing: .7px;
+  border: 1px solid $blue;
+}
+
+.button-base {
+  color: #fff;
+  text-transform: uppercase;
+  background-color: $blue;
+}
+
+.button--outline {
+  background-color: transparent;
+  border: 1px solid $blue;
+  color: $darker-blue;
+}
+
+.button-redo {
+  position: relative;
+  padding: .3em 2.8em .15em 1.5em;
+  &:after {
+    position: absolute;
+    content: '';
+    background-image: url(../graphics/layout/arrow-right.svg);
+    background-size: 11px 11px;
+    width: 11px;
+    height: 11px;
+    background-repeat: no-repeat;
+    display: inline-block;
+    right: .8em;
+    top: .6em;
+  }
+}
+
+.button-undo {
+  position: relative;
+  margin-right: .8em;
+  padding: .3em 1.5em .15em 2.8em;
+  &:before {
+    position: absolute;
+    content: '';
+    background-image: url(../graphics/layout/arrow-left.svg);
+    background-size: 11px 11px;
+    width: 11px;
+    height: 11px;
+    padding-right: .5em;
+    background-repeat: no-repeat;
+    display: block;
+    display: inline-block;
+    left: .8em;
+    top: .6em;
+  }
+}
+
+.select-wrapper {
+  position: relative;
+  display: inline-block;
+  &:after {
+    position: absolute;
+    content: '';
+    background-image: url(../graphics/layout/chevron-down.svg);
```

Comment:
Probably requires a JSX change since it seems like this should be on the wrapper